src/main/java/com/company/Main.java

```diff
@@ -12,14 +12,17 @@ public class Main {
     public static void main(String[] args) throws IOException {
         Map<String, List<Integer>> companies = new HashMap<>();
         List<Integer> list = new ArrayList<>();
```

Comment:
to jest lista czego? bo nazwa nic nie mowi. Plus fajnie gdyby to nie byly inty. Ale to juz szczegol